lib/index.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+const { logger } = require('express-wolox-logger');
+const { printSchema } = require('graphql');
+const { info } = require('log-symbols');
+const { args } = require('./helper/commandParser');
+const { getChangesInSchema, logChanges } = require('./helper/schemas');
+const { resolveInspection } = require('./resolvers/inspector');
+const {
+  status,
+  argsNames: { OLD_SCHEMA, NEW_SCHEMA, IGNORE_OLD_SCHEMA }
+} = require('./constants');
+
+const processcwd = process.cwd();
+const oldSchemaFile = `${processcwd}/${args[OLD_SCHEMA]}`;
+const { schema } = require(`${processcwd}/${args[NEW_SCHEMA]}`);
+const newSchema = printSchema(schema);
+
+try {
+  logger.info(info, 'Schema inspection:');
+  let changes = [];
+  if (fs.existsSync(oldSchemaFile)) {
+    changes = getChangesInSchema(oldSchemaFile, newSchema);
```

Comment:
You could use const in changes and do here something like `changes.push(...getChangesInSchema(oldSchemaFile, newSchema))`